packages/components/src/calendar-react-day-picker/localization.ts

```diff
@@ -0,0 +1,138 @@
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import { useMemo } from 'react';
+import type { Modifiers, BaseProps } from './types';
+
+function isLocaleRTL( localeCode: string ) {
```

Comment:
Another idea is that [`Intl.Locale#getTextInfo`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getTextInfo) would give us this information as well. Unfortunately, it's not supported in Firefox, so probably not an option for now.

i.e.

```suggestion
const isLocaleRTL = ( localeCode: string ) =>
	new Intl.Locale( localeCode ).getTextInfo().direction === 'rtl';
```

Reply:
Good idea, I can use that function if available, and otherwise fallback to manual logic